.github/pull_request_template.md

```diff
@@ -1,7 +1,42 @@
 # Description
 
 [Provide a brief description of the changes introduced by this pull request. Explain the purpose and goals of the changes.]
+[Please include a summary of the changes and the motivation behind them.]
```

Comment:
NIT: Do we need both of these lines? They seem to be repeating the same request.

Reply:
Yeah sure , if you want we can remove line 4.